Assets/Xbox Live/Scripts/UserProfile.cs

```diff
@@ -328,13 +252,36 @@ public static Color ColorFromHexString(string color)
         return new Color(r, g, b);
     }
 
+    private void SetupForRefresh(XboxLiveUser xboxLiveUser, XboxLiveAuthStatus authStatus, string errorMessage)
```

Comment:
Might be useful to some how print the error message - as it'll never be surfaced up to the developer. If it is friendly, it may be something to print to the user.

Reply:
Done everywhere. Will be in the next push.